src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Subscriber.kt

```diff
@@ -0,0 +1,51 @@
+package com.bardsoftware.backend.fleet.rmarkdown
+
+import com.google.cloud.ServiceOptions
+import com.google.cloud.pubsub.v1.AckReplyConsumer
+import com.google.cloud.pubsub.v1.MessageReceiver
+import com.google.cloud.pubsub.v1.Subscriber
+import com.google.pubsub.v1.PubsubMessage
+import com.google.pubsub.v1.SubscriptionName
+import java.util.concurrent.LinkedBlockingDeque
+
+object SubscriberExample {
+    private const val subscriberName = "rmarkdown-compiler"
```

Comment:
Please pass subscription name in the command line arguments. You can parse command line with [this library](https://github.com/xenomachina/kotlin-argparser)